highlighting/lib/src/extension/compiler_extensions.dart

```diff
@@ -14,8 +16,9 @@ void skipIfHasPrecedingDot(DomainRegexMatch match, Response response) {
 }
 
 void scopeClassName(Mode mode, [Mode? parent]) {
```

Comment:
Looks like all calls to this have `parent`.
```suggestion
void scopeClassName(Mode mode, Mode? parent) {
```